src/integrations/executable.rs

```diff
@@ -0,0 +1,6 @@
+use crate::models::resume::Resume;
+use std::error::Error;
+
+pub(crate) trait Executable {
```

Comment:
Naming conventions in Rust seem to avoid the "able" suffix, so the trait should probably be named "Execute".

Reply:
+1